mozilla/bedrock
# [bug 1422899] Remove old dev edition product page

## Description
- Removes old firefox developer product page template
- Moves dev edition /firstrun and /whatsnew templates to the same directory, as they share much of the same code (and lang files).

Note: I have left the old dev edition /whatsnew and /firstrun templates, as I'm not sure it makes sense to show the quantum template to older versions.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1422899

## Testing
- Probably a good idea to run gulp static:clean prior to testing this PR.
- Check I didn't remove any other assets that may still be needed?
- Check both the /firstrun and /whatsnew pages still look as expected.
- Check translations still appear as expected.
